src/plugins/calendar/calendar.js

```diff
@@ -324,6 +326,9 @@ $document.on( "click vclick touchstart", ".cal-month-prev, .cal-month-next", fun
 		modifier = className.indexOf( "cal-month-prev" ) !== -1 ? -1 : 1,
 		date = new Date( calendar.lib.year, calendar.lib.month + modifier, 1 );
 
+	// Added declaration of current month in aria-live="polite" for screen-readers
+	$calendar.find( ".current-month" ).html( i18nText.monthNames[ date.getMonth() ] + " " + calendar.lib.year );
```

Comment:
The "current-month" span are not updated when the user change the month or the year directly via the select elements.
Idem when we navigate with the days with the "left" and "right" arrow and the current month change

You might need to hook other event and apply the same/similar update to the current-month.